docs/user-guide/third-party-addons/opentelemetry.md

```diff
@@ -191,7 +191,7 @@ To install the example and collectors run:
     helm repo add grafana https://grafana.github.io/helm-charts
     helm repo update
     # deploy cert-manager needed for OpenTelemetry collector operator
-    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.13.3/cert-manager.yaml
```

Comment:
Can you explain choosing 1.13.3 of cert-manager?

Reply:
This was the latest version when I initially started working on this PR.

Reply:
I updated it to v1.15.3 now.